lib/accession/accession/tag.rb

```diff
@@ -58,5 +58,53 @@ def array_express_label
     def attributes
       %i[services value name groups ebi_name]
     end
+
+    # Some helper methods for displaying information from tags
+    module HelperTagValue
+      NOT_COLLECTED = 'not collected'
+      NOT_PROVIDED = 'not provided'
+      RESTRICTED_ACCESS = 'restricted access'
+      OTHER_DEFAULT_SETTINGS = [NOT_COLLECTED, NOT_PROVIDED, RESTRICTED_ACCESS].freeze
```

Comment:
Check if 'not applicable' is necessary. https://ena-docs.readthedocs.io/en/latest/submit/samples/missing-values.html